internal/server/iigointernal/judiciary.go

```diff
@@ -132,3 +151,140 @@ func softMergeSanctionThresholds(clientSanctionMap map[roles.IIGOSanctionTier]ro
 	}
 	return defaultMap
 }
+
+// getIslandSanctionTier if statement based evaluator for which sanction tier a particular score is in
+func getIslandSanctionTier(islandScore roles.IIGOSanctionScore, scoreMap map[roles.IIGOSanctionTier]roles.IIGOSanctionScore) roles.IIGOSanctionTier {
+	if islandScore <= scoreMap[roles.SanctionTier1] {
+		return roles.NoSanction
+	} else if islandScore <= scoreMap[roles.SanctionTier2] {
```

Comment:
Shouldn't this be `<` not `<=`. Id assume if someone gets 5 points they should get a sanction of tier 1

Reply:
Hmmm good point, could this be in another pr as well (it s a bit of risky business rebasing the following commits)

Reply:
It's fine rebasing seems unavoidable